src/Webapi/Dom/Webapi__Dom__Document.res

```diff
@@ -25,6 +25,8 @@ module Impl = (
   let ofNode = (node: Dom.node): option<T.t> =>
     Webapi__Dom__Node.nodeType(node) == Webapi__Dom__Types.Document ? Some(Obj.magic(node)) : None
 
+  @get @return(nullable) external activeElement: T.t => option<Dom.element> = "activeElement"
+  @set external setActiveElement: (T.t, Dom.node_like<'a>) => unit = "activeElement"
```

Comment:
This property is readonly you change it by calling focus. https://developer.mozilla.org/en-US/docs/Web/API/Document/activeElement

- [ ] Remove setter for activeElement

Reply:
that came from our internal bindings 😂 oops